raw_action.go

```diff
@@ -11,22 +11,25 @@ import (
 
 type RawAction struct {
 	BaseAction `yaml:",inline"`
+	Origin     string // there te source comes from
 	Offset     string
-	Source     string
-	Path       string
+	Source     string // relative path inside of origin
 }
 
 func (raw *RawAction) Verify(context *DebosContext) error {
-	if raw.Source != "filesystem" {
-		return errors.New("Only suppport sourcing from filesystem")
+	if len(raw.Origin) <= 0 || len(raw.Source) <= 0 {
```

Comment:
A string can't be smaller then 0 :) so that's an odd check

Reply:
Pretty good old habbit ;)
No influence to generated code so I'd left it as is.

Reply:
Yeah but it's weird for the reader so better change it :)

Reply:
done